blockstore/src/main/java/brooklyn/location/blockstore/BlockDeviceOptions.java

```diff
@@ -12,7 +12,7 @@
     private String name;
     private String zone;
     private Map<String, String> tags = Maps.newHashMap();
-    private int sizeInGb;
+    private Number sizeInGb;
```

Comment:
Why change it to a Number if you actually expect it to be an Integer (according to the error message)? Could instead cast whatever you have to int.